### PR TITLE
Add CSI Ephemeral inline volume docs

### PR DESCRIPTION
**Pull request by ggriffiths**
Signed-off-by: Grant Griffiths <grant@portworx.com>

### CODE REVIEW

File: content/portworx-install-with-kubernetes/storage-operations/csi/_index.md [hunk 1]
```diff
@@ -352,6 +355,36 @@ You can clone CSI-enabled volumes, duplicating both the volume and content withi
       kubectl apply -f clonePVC.yaml
       ```
 
+## Ephemeral Inline Volumes with CSI
+To use an ephemeral inline volume with the Portworx CSI driver, declare it inside of the pod spec as seen below.
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/storage-operations/csi/_index.md]: Can we present this in steps like we did for the sections above? 
1. enable feature gate
2. create a pod spec, specifying ???
3. apply the spec to create ???

File: content/portworx-install-with-kubernetes/storage-operations/csi/_index.md [hunk 2]
```diff
@@ -17,16 +17,22 @@ Using Portworx with CSI, you can perform the following operations:
...
+| [CSI Snapshots](/portworx-install-with-kubernetes/storage-operations/csi/#take-snapshots-of-csi-enabled-volumes) | 2.2 | 1.16 | 1.13.12 |
+| Stork | 2.2 | 1.13 |  |
+| Volume Expansion (resizing) | 2.2 | 1.16 | 1.14 |
+| [Ephemeral Inline Volumes](/portworx-install-with-kubernetes/storage-operations/csi/#ephemeral-inline-volumes-with-csi) | 2.5 | 1.16 | 1.15 |
 
-[^1]: Note that only Stork 2.3.0 or later is supported with CSI.
+{{<info>}}
+**NOTE:**
+
+* Note that only Stork 2.3.0 or later is supported with CSI.
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/storage-operations/csi/_index.md]: @ggriffiths the stork footnote wasn't working for me, so I wrapped them both under a note list.

File: content/portworx-install-with-kubernetes/storage-operations/csi/_index.md [hunk 3]
```diff
@@ -352,6 +358,46 @@ You can clone CSI-enabled volumes, duplicating both the volume and content withi
...
+        driver: pxd.portworx.com
+        volumeAttributes:
+          size: "2Gi"
+```
+
+Ephemeral inline volumes may also be used with deployments and other application types. See this [Kubernetes blog](https://kubernetes.io/blog/2020/01/21/csi-ephemeral-inline-volumes/) for more information and example use cases.
+
+
+### Ephemeral Inline Volumes alpha
+To enable the alpha version of this feature in Kubernetes, enable the `CSIInlineVolume` feature gate in Kubernetes 1.15:
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/storage-operations/csi/_index.md]: Why do mention alpha here? for solutions architects and prototyping?

Comment (ggriffiths) [reply content/portworx-install-with-kubernetes/storage-operations/csi/_index.md]: Yes mainly - I can remove this if it adds confusion.

File: content/portworx-install-with-kubernetes/storage-operations/csi/_index.md (hunk 2 above)
Comment (ggriffiths) [reply content/portworx-install-with-kubernetes/storage-operations/csi/_index.md]: thanks :+1:

File: content/portworx-install-with-kubernetes/storage-operations/csi/_index.md (hunk 1 above)
Comment (ggriffiths) [reply content/portworx-install-with-kubernetes/storage-operations/csi/_index.md]: No feature gate is needed, but yes I'll update for steps 2 and 3

Comment (ggriffiths) [reply content/portworx-install-with-kubernetes/storage-operations/csi/_index.md]: Updated with steps.

File: content/portworx-install-with-kubernetes/storage-operations/csi/_index.md (hunk 3 above)
Comment (ggriffiths) [reply content/portworx-install-with-kubernetes/storage-operations/csi/_index.md]: Removed this

File: content/portworx-install-with-kubernetes/storage-operations/csi/_index.md [hunk 4]
```diff
@@ -17,14 +17,17 @@ Using Portworx with CSI, you can perform the following operations:
...
+| Feature | Portworx version | Supported Kubernetes version | Alpha Kubernetes version |
+| --- | --- | --- | --- |
+| [Provision, attach, and mount volumes](/portworx-install-with-kubernetes/storage-operations/csi/#create-and-use-persistent-volumes) | 2.2 | 1.13.12 |  |
+| [CSI Snapshots](/portworx-install-with-kubernetes/storage-operations/csi/#take-snapshots-of-csi-enabled-volumes) | 2.2 | 1.16 | 1.13.12 |
+| Stork [^1] | 2.2 | 1.13 |  |
+| Volume Expansion (resizing) | 2.2 | 1.16 | 1.14 |
+| [Ephemeral Inline Volumes](/portworx-install-with-kubernetes/storage-operations/csi/#ephemeral-inline-volumes-with-csi) | 2.5 | 1.16 | 1.15 |
+
+It is not recommended to use alpha Kubernetes features in production as the API and core functionality is not finalized. Users that adopt alpha features in production may need to perform costly manual upgrades.
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/storage-operations/csi/_index.md:30]:
```suggestion
Portworx, Inc. does not recommend that you use alpha Kubernetes features in production as the API and core functionality are not finalized. Users that adopt alpha features in production may need to perform costly manual upgrades.
```

File: content/portworx-install-with-kubernetes/storage-operations/csi/_index.md [hunk 5]
```diff
@@ -352,6 +355,39 @@ You can clone CSI-enabled volumes, duplicating both the volume and content withi
       kubectl apply -f clonePVC.yaml
       ```
 
+## Ephemeral Inline Volumes with CSI
+You can create ephemeral inline volumes with CSI. These volumes are created during pod creation and deleted upon pod teardown.
+
+1. Create a pod spec that uses the Portworx CSI driver, declaring the inline volume as seen below in a YAML file named `ephemeral-volume-pod.yaml`
+```
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/storage-operations/csi/_index.md:362]:
```suggestion
```text
```

File: content/portworx-install-with-kubernetes/storage-operations/csi/_index.md [hunk 6]
```diff
@@ -352,6 +355,39 @@ You can clone CSI-enabled volumes, duplicating both the volume and content withi
...
+    - name: my-csi-volume
+      csi:
+        driver: pxd.portworx.com
+        volumeAttributes:
+          size: "2Gi"
+```
+
+2. Apply the `ephemeral-volume-pod.yaml`  spec to create the pod with an ephemeral inline volume:
+
+      ```text
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/storage-operations/csi/_index.md:385]: note for previous comment, do it just like this.

File: content/portworx-install-with-kubernetes/storage-operations/csi/_index.md [hunk 7]
```diff
@@ -352,6 +355,39 @@ You can clone CSI-enabled volumes, duplicating both the volume and content withi
...
+      kubectl apply -f ephemeral-volume-pod.yaml
+      ```
+
+Ephemeral inline volumes may also be used with deployments and other application types. See this [Kubernetes blog](https://kubernetes.io/blog/2020/01/21/csi-ephemeral-inline-volumes/) for more information and example use cases.
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/storage-operations/csi/_index.md:389]:
```suggestion
You may also use ephemeral inline volumes with deployments and other application types. See this [Kubernetes blog](https://kubernetes.io/blog/2020/01/21/csi-ephemeral-inline-volumes/) for more information and example use cases.
```

**Pull request merged**